stubs/setupVars.conf

```diff
@@ -0,0 +1,3 @@
+piholeInterface=eth0
+IPv4_address=192.168.1.100/24
+IPv6_address=fe80::bc3b:7855:7c13:4f84
```

Comment:
Even in your new code you look for the new names (e.g. [here](https://github.com/pi-hole/AdminLTE/pull/219/files#diff-358d439e94c8f14050244e4012f5dacaR81))